src/config/validation/taskSchema.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from "zod";
+
+export const taskSchema = z.object({
+    id: z.string().optional(),
```

Comment:
this can be
z.string().uuid() if I'm not wrong

Reply:
You're right, I'll change it